packages/expo-updates/ios/EXUpdates/Update/RawManifest/EXUpdatesNewRawManifest.h

```diff
@@ -10,6 +10,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (NSString *)createdAt;
 - (NSString *)runtimeVersion;
 - (NSDictionary *)launchAsset;
+- (nullable NSArray *)assets;
```

Comment:
objc supports generics - added for better swift bridging - worth using them here i think: https://developer.apple.com/documentation/swift/imported_c_and_objective-c_apis/using_imported_lightweight_generics_in_swift

Reply:
I'll look into generics for both Android and iOS in a follow-up which should help us ensure consistency.